### DISCUSSION TITLE
Player View and Smart Home Streaming

### DISCUSSION THREAD

**Discussion by Thunderhawk2001** [XilinJia/Podcini#93] [General]
Hello :)

Just wanted to ask if Im just blind or is there no way to show a "full screen player view"  linke in Antennapod?
Like if I tap on the player controls on the bottom, it opens a full screen view of the currently played podcast. In this view I can also tap on the "Streaming" button to select my Google Home speakers and speaker groups to play the podcast in m home.

First one I could live without, but the second is the biggest deal breaker for me at the moment, as I like to start listening at home and then continue in my car on my way to work.

**Comment by XilinJia** [comment]
I haven't tried, but I think the cast to speaker function is in the "Play" version, like in AP. So you should download the Play apk, rather than the free one. Let me know how that works.

**Reply by Thunderhawk2001** [reply]
Yes! That's it. Thank you.

**Reply by XilinJia** [reply]
Good. You said "First one I could live without, but the second is the biggest deal breaker", which is which?  Did you get Podcini to work on the car?

**Reply by Thunderhawk2001** [reply]
Oh, sorry. That wasn't very specific. 1st was "full-screen player view" and 2nd was "smart home streaming".
I can live without a dedicated full-screen player, just need to get accustomed it. Mostly need it for the button for "smart home streaming", as this button is not available in the player controls on the bottom.

Yes, as instructed in the AndroidAuto.md = after enabling "other sources" in Android Auto, I was able to use it.

**Reply by XilinJia** [reply]
explain: a dedicated full-screen player

in Podcini, when you tap on the icon in the bottom player, the player expands to full screen, is that not what you needed?

**Reply by Thunderhawk2001** [reply]
Sure, will try:

e.g. If you tap in Spotify on the bottom player you get a "full-screen player". Showing just an image of the song/band/album/podcast, the player controls and on the bottom left the streaming button:

![Screenshot 2024-09-24 131229 - Copy](https://github.com/user-attachments/assets/02ecaf04-000e-449c-9b89-4b64cb1f9cad)


in Podcini you will see the podcast description. Just talking about me, but I personally don't need the podcast description in this location. I get why it's there and I can accept it. That's why I said, that I can live with that, as long as the streaming button is there (which it is)

![Screenshot 2024-09-24 131453 - Copy](https://github.com/user-attachments/assets/975251db-45b5-44ec-9379-31d25f4af1a9)


I hope you don't take that the wrong way, because I don't want to discriminate you or your awesome work here. It's just my personal preference.

**Reply by XilinJia** [reply]
No, I take your comments quite positively.
I changed the full player view with the description at the beginning because I thought that's more important than a plain image, especially when the description is some kind of a transcript coming with some podcasts. The plain image seems nearly useless to me (voice your different opinion), but it can be found at the end if someone wants to see it.

**Reply by Thunderhawk2001** [reply]
Ok, then I can stop worrying.
You are right, it is pretty useless, but looks "good" ^^ I'll give this version a try. Maybe I'll like it better, after using it for a bit. It's just unfamiliar, as I never really looked at the podcast descriptions, which would be quite dangerous while driving ;)

But if it's not to much work, maybe there could be a toggle between useless/informative ?
